src/main/java/org/openrewrite/java/testing/mockito/MockitoWhenOnStaticToMockStatic.java

```diff
@@ -195,14 +199,17 @@ public J.ClassDeclaration visitClassDeclaration(J.ClassDeclaration classDecl, Ex
                                 .build()
                                 .apply(updateCursor(classDecl), classDecl.getBody().getCoordinates().firstStatement());
 
-                        if (classDecl.getBody().getStatements().stream().noneMatch(it -> isMethodDeclarationWithAnnotation(it, AFTER, AFTER_CLASS))) {
-                            Optional<Statement> beforeMethodJunit = after.getBody().getStatements().stream()
-                                    .filter(it -> isMethodDeclarationWithAnnotation(it, JUNIT_ANNOTATION))
+                        if (classDecl.getBody().getStatements().stream().noneMatch(it -> isMethodDeclarationWithAnnotation(it, AFTER))) {
```

Comment:
Code looks fine to me! But maybe you can use some functions to remove the duplication (only if it really cleans things up).